### PR TITLE
Replace refs with tags for 2.7.0 manifests and remove cron

### PR DESCRIPTION
**Pull request by zelinh**
### Description
Replace refs with tags for 2.7.0
Remove cron build job for 2.6.1
Remove cron perf-test job for 2.5.0

### Issues Resolved
Part of #3230 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: jenkins/opensearch/perf-test.jenkinsfile [hunk 1]
```diff
@@ -25,15 +25,6 @@ pipeline {
...
-            H 21 * * * %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.5.0/latest/linux/arm64/tar/dist/opensearch/manifest.yml;TEST_WORKLOAD=http_logs;TEST_ITERATIONS=3;WARMUP_ITERATIONS=2
-            H 21 * * * %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.5.0/latest/linux/arm64/tar/dist/opensearch/manifest.yml;TEST_WORKLOAD=http_logs;TEST_ITERATIONS=250;WARMUP_ITERATIONS=0
-            H 21 * * 5 %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.5.0/latest/linux/x64/tar/dist/opensearch/manifest.yml;TEST_WORKLOAD=http_logs;TEST_ITERATIONS=3;WARMUP_ITERATIONS=2
-            H 21 * * 5 %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.5.0/latest/linux/x64/tar/dist/opensearch/manifest.yml;TEST_WORKLOAD=http_logs;TEST_ITERATIONS=250;WARMUP_ITERATIONS=0
-
             H 19 * * * %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.6.0/latest/linux/arm64/tar/dist/opensearch/manifest.yml;TEST_ITERATIONS=3;WARMUP_ITERATIONS=2
```
Comment (gaiksaya) [inline jenkins/opensearch/perf-test.jenkinsfile:28]: Please remove 2.6.0 as well

**Review by gaiksaya — CHANGES_REQUESTED** [review]
@bbarani Can we remove 1.3.9 performance test runs as well?

File: jenkins/opensearch/perf-test.jenkinsfile (hunk 1 above)
Comment (zelinh) [reply jenkins/opensearch/perf-test.jenkinsfile:28]: We would need 1.3.9 perf for upcoming 1.3.10 release right? @bbarani

Comment (gaiksaya) [reply jenkins/opensearch/perf-test.jenkinsfile:28]: We have numbers from previous runs so should be good. Waiting on reply from @bbarani Feel free to remove 2.6.0 till then,

Comment (bbarani) [reply jenkins/opensearch/perf-test.jenkinsfile]: Can we add 2.8.0 to the list? @zelinh

Comment (bbarani) [reply jenkins/opensearch/perf-test.jenkinsfile]: Lets remove 1.3.9 and add 1.3.10.

**Review by peterzhuamazon — APPROVED** [review]

**Review by gaiksaya — APPROVED** [review]

File: jenkins/opensearch/perf-test.jenkinsfile (hunk 1 above)
Comment (zelinh) [reply jenkins/opensearch/perf-test.jenkinsfile]: We would have a separate PR to add 1.3.10 and remove 1.3.9 once the release cycle for 1.3.10 starts.

**Comment by zelinh** [comment]
The CI failure is not related to this PR. Merging this for now.

**Pull request merged**